src/test/java/httpServer/RequestBuilderTest.java

```diff
@@ -0,0 +1,72 @@
+package httpServer;
+
+import httpServer.http.request.HTTPRequest;
+import httpServer.http.request.HTTPRequestBuilder;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+import static httpServer.http.Constants.CRLF;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class RequestBuilderTest {
```

Comment:
It's a little confusing because the class is called `HTTPRequestBuilder` but the test is called `RequestBuilderTest`.